rotkehlchen/exchanges/binance.py

```diff
@@ -61,6 +62,7 @@
 # https://www.binance.com/en/support/articles/115000599831-Binance-Exchange-Launched-Date-Set
 BINANCE_LAUNCH_TS = Timestamp(1500001200)
 API_TIME_INTERVAL_CONSTRAINT_TS = Timestamp(7776000)  # 90 days
+BINANCE_LENDING_INTEREST_HISTORY_INTERVAL_TS = Timestamp(2592000)  # 30 days
```

Comment:
Point me out if I'm wrong but interval is a timestamp given that it's a duration -- timedelta in python terms.
Or did you mean something else?

Reply:
Timestamp is a type that we have defined and semantically is a moment of time in seconds. Since is a type over int it won't have implications code wise but for me at least is strange to see a difference in seconds as a Timestamp since it doesn't mean a moment in time